src/docs/app-monitoring/sysdig-monitor-create-alert-channels.md

```diff
@@ -21,85 +21,89 @@ sort_order: 3
 # Create alerts and notifications in Sysdig Monitor
 Last updated: **May 31, 2023**
 
-You can create alerts based on monitoring dashboards in Sysdig Monitor, that notify your team when something needs attention.
+You can create alerts based on monitoring dashboards in Sysdig Monitor that notify your team when something needs attention.
 
-Here are some steps on how to setup the Sysdig alerts with [Rocket.Chat](https://chat.developer.gov.bc.ca/).
+Here are some steps on how to set up Sysdig alerts with [Rocket.Chat](https://chat.developer.gov.bc.ca/).
 
 ## On this page
 - [Create alerts and notifications in Sysdig Monitor](#create-alerts-and-notifications-in-sysdig-monitor)
-- [Create a Rocket.Chat chat channel and  webhook for alert messages](#create-a-rocketchat-chat-channel-and--webhook-for-alert-messages)
-- [Create a Sysdig team notification channel](#create-a-sysdig-team-notification-channel)
-- [Creating an Alert](#creating-an-alert)
-- [Related pages](#related-pages)
+  - [On this page](#on-this-page)
+  - [Create a Rocket.Chat chat channel and webhook for alert messages](#create-a-rocketchat-chat-channel-and-webhook-for-alert-messages)
+  - [Create a Sysdig team notification channel](#create-a-sysdig-team-notification-channel)
+  - [Creating an Alert](#creating-an-alert)
+  - [Related pages:](#related-pages)
+  - [Related resources:](#related-resources)
 
 <!-- ### End of On this page -->
 
-## Create a Rocket.Chat chat channel and  webhook for alert messages
+## Create a Rocket.Chat chat channel and webhook for alert messages
 
 Rocket.Chat requires an incoming webhook to parse the data from Sysdig.
 
-1.  Create a RC chat channel for the alert messages to arrive if there isn't one
+1. Create a RC chat channel for the alert messages to arrive if there isn't one.
 
-2. Create an incoming webhook to the chat channel:
-  -  Click on your avatar -> Administration -> Integrations -> New (from Incoming tab)
+   **Note:** You can do this by clicking on the 'Create New' menu in the left navigation pane and selecting `New Channel`.
+    ![RC webhook config](../../images/sysdig-team-rc-create-channel.png)
 
-  - Name the webhook in the format of `sysdig-alert-webhook-<APP_TEAM_NAME>`
+2. Create an incoming webhook for the chat channel:
 
+   - Click on ellipsis menu icon-> Workspace -> Integrations -> New (on the right top corner). or at https://chat.developer.gov.bc.ca/admin/integrations -> New
```

Comment:
```suggestion
   - In the left navigation pane of Rocketchat, click on the ellipsis menu icon-> Workspace -> Integrations -> New (on the right top corner). or at https://chat.developer.gov.bc.ca/admin/integrations -> New
```
Clarifying here since rocketchat has more than one elipses menu icon.